app_dart/lib/src/request_handlers/check_flaky_tests_and_update_github.dart

```diff
@@ -185,7 +188,7 @@ class CheckForFlakyTestAndUpdateGithub extends ApiRequestHandler<Body> {
     return target != null && target[_ciYamlTargetIsFlakyKey] == true;
   }
 
-  YamlList _getTags(String builderName, YamlMap ci) {
+  List<dynamic> _getTags(String builderName, YamlMap ci) {
```

Comment:
Shall we use String explicitly? do we have other type other than string

Reply:
I don't know whether there is plan to extend to usage of tags. The problem with forcing it to be string is that we may break it without knowing it if we change it later since the test is using mock data not the real ci.yaml in master branch.